src/components/ConnectionLayout.tsx

```diff
@@ -29,9 +29,27 @@ type ConnectionLayoutProps = {
     accessVariants?: PolicyAccessVariant[];
     /** The current feature name that the user tries to get access to */
     featureName?: PolicyFeatureName;
+    /** The content container style of Scrollview */
+    contentContainerStyle?: StyleProp<ViewStyle> | undefined;
+    /** Style of the title text */
+    titleStyle?: StyleProp<TextStyle> | undefined;
+    /** Style of the subtitle text */
```

Comment:
@hungvu193 please add a space above all comments
```suggestion
    
    /** Style of the subtitle text */
```

Reply:
Updated!